src/com/lilithsthrone/controller/MainController.java

```diff
@@ -5148,6 +5159,88 @@ public void effects() {
 			}
 		}
 		
+		// Save/load enchantment:
+		if (Main.game.getCurrentDialogueNode() == EnchantmentDialogue.ENCHANTMENT_SAVE_LOAD) {
+			for (File f : EnchantmentDialogue.getSavedEnchants()) {
+				id = "overwrite_saved_" + f.getName().substring(0, f.getName().lastIndexOf('.'));
```

Comment:
you make use of "f.getName().substring(0, f.getName().lastIndexOf('.'))" a lot here. Recommend assigning to local variable.